design-proposals/klip-23-primary-keys-for-tables.md

```diff
@@ -0,0 +1,161 @@
+# KLIP 23 - Use PRIMARY KEYs for tables
+
+**Author**: @big-andy-coates |
+**Release Target**: TBD |
+**Status**: In Discussion |
+**Discussion**: TBD
+
+**tl;dr:** Tables in SQL have `PRIMARY KEY`s. In ksqlDB, Tables, like Streams currently have only
+`KEY` columns, i.e. columns that come from the Kafka message key. We should introduce `PRIMARY KEY`
```

Comment:
Effectively, the key (from the topic) of a table is already a primary key for the table. This sentence implies it's not a primary key and we want to change this. It might be worth to rephrase to make clear that this KLIP is only a syntax change but not a semantic change? (Ie, we bring the syntax up to data to match the existing semantics.)

Reply:
done.